docs/features/pytorch.md

```diff
@@ -0,0 +1,140 @@
+
+=== "`pixi.toml`"
+    ```toml title="Adding a cpu environment"
+    --8<-- "docs/source_files/pixi_tomls/pytorch-conda-forge.toml:use-envs"
+    ```
+=== "`pyproject.toml`"
+    ```toml title="Split into environments and add a CPU environment"
+    --8<-- "docs/source_files/pyproject_tomls/pytorch-conda-forge.toml:use-envs"
+    ```
+
+Now you should be able to extend that with your dependencies and tasks.
+
+Links to notable packages:
+
+- [pytorch](https://prefix.dev/channels/conda-forge/packages/pytorch)
+- [pytorch-cpu](https://prefix.dev/channels/conda-forge/packages/pytorch-cpu)
+- [pytorch-gpu](https://prefix.dev/channels/conda-forge/packages/pytorch-gpu)
+- [torchvision](https://prefix.dev/channels/conda-forge/packages/torchvision)
+- [torchaudio](https://prefix.dev/channels/conda-forge/packages/torchaudio)
+- [cuda-version](https://prefix.dev/channels/conda-forge/packages/cuda-version)
+
+## Installing from PyPi
+Thanks to the integration with `uv` we can also install PyTorch from `pypi`.
+
+### Pytorch index
+PyTorch packages are provided through a custom index, which is maintained by the PyTorch team.
+To install PyTorch from the PyTorch index, you need to add the indexes to manifest.
+Best to do this per dependency to force the index to be used.
+
+- CPU only: [https://download.pytorch.org/whl/cpu](https://download.pytorch.org/whl/cpu)
+- CUDA 11.8: [https://download.pytorch.org/whl/cu118](https://download.pytorch.org/whl/cu118)
+- CUDA 12.1: [https://download.pytorch.org/whl/cu121](https://download.pytorch.org/whl/cu121)
+- CUDA 12.4: [https://download.pytorch.org/whl/cu124](https://download.pytorch.org/whl/cu124)
+- ROCm6: [https://download.pytorch.org/whl/rocm6.2](https://download.pytorch.org/whl/rocm6.2)
+
+=== "`pixi.toml`"
+    ```toml title="Install PyTorch from pypi"
+    --8<-- "docs/source_files/pixi_tomls/pytorch-pypi.toml:minimal"
+    ```
+=== "`pyproject.toml`"
+    ```toml title="Install PyTorch from pypi"
+    --8<-- "docs/source_files/pyproject_tomls/pytorch-pypi.toml:minimal"
+    ```
+
+## Installing from PyTorch channel
+!!! warning
+    This depends on the [non-free](https://www.anaconda.com/blog/is-conda-free) `main` channel from Anaconda and mixing it with `conda-forge` can lead to conflicts.
+
+!!! note
+    This is the [legacy](https://dev-discuss.pytorch.org/t/pytorch-deprecation-of-conda-nightly-builds/2590) way of installing pytorch, this will not be updated to later versions as pytorch has discontinued their channel.
+
+=== "`pixi.toml`"
+    ```toml title="Install PyTorch from the PyTorch channel"
+    --8<-- "docs/source_files/pixi_tomls/pytorch-from-pytorch-channel.toml:minimal"
+    ```
+=== "`pyproject.toml`"
+    ```toml title="Install PyTorch from the PyTorch channel"
+    --8<-- "docs/source_files/pyproject_tomls/pytorch-from-pytorch-channel.toml:minimal"
+    ```
+
+## Troubleshooting
+
+- You can test the installation with the following command:
```

Comment:
Looking at the rendered version I think using ### or #### instead of bullets would be nicer.

Reply:
I currently wrote it as a summary of the implementations. So that's why I put it on the main level of the document which is `##`. We could split it up but I mainly added this to allow search to find this file when users have these issues.

Reply:
Not sure if I misunderstand but I mean the bullets under the `## Troubleshooting`. So you would get something more like:

```
## Troubleshooting

#### How can I test the installation?

```

And maybe make all of them questions? Like more of FAQ style, wdyt?

Reply:
Ohh that might look nicer indeed. Let me try that.

Reply:
Changed that! Looks much better!